src/shared/components/palette/case-flag/components/search-language-interpreter/search-language-interpreter.component.html

```diff
@@ -0,0 +1,35 @@
+<h2 class="govuk-heading-l">Search for a language interpreter</h2>
+<div class="form-group" [formGroup]="formGroup">
+  <div class="auto-complete-container">
+    <p>Enter the language that will need to be interpreted. If this language is not listed, you can enter it manually.</p>
+    <input
+      [formControl]="formGroup.controls.searchTerm"
+      [matAutocomplete]="autoSearchLanguage"
+      class="govuk-input">
+    <mat-autocomplete class="mat-autocomplete-panel-extend" autoActiveFirstOption #autoSearchLanguage="matAutocomplete">
+      <mat-option *ngFor="let language of filteredLanguages$ | async" [value]="language">
+        {{language}}
+      </mat-option>
+      <!-- <mat-option *ngIf="!languages.length && showAutocomplete && term && term.length >= this.minSearchCharacters">No results found</mat-option> -->
```

Comment:
@johnbenjamin-hmcts - perhaps this commented-out line should be removed.

Reply:
@Daniel-Lam - I think this might be required when we integrate to the reference data, but not in the design at the moment.
This is to show no results found message to the user.

Reply:
Fixed as part of the base branch https://github.com/hmcts/ccd-case-ui-toolkit/pull/1018